tests/phpunit/integration/Enforce_Policies_Tests.php

```diff
@@ -0,0 +1,369 @@
+<?php
+/**
+ * Class Google\WP_Fetch_Metadata\Tests\PHPUnit\Integration\Enforce_Policies_Tests
+ *
+ * @package   Google\WP_Fetch_Metadata
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://wordpress.org/plugins/fetch-metadata
+ */
+
+namespace Google\WP_Fetch_Metadata\Tests\PHPUnit\Integration;
+
+use Google\WP_Fetch_Metadata\Tests\PHPUnit\Framework\Integration_Test_Case;
+use Google\WP_Fetch_Metadata\Policy\Isolation_Policy;
+use Google\WP_Fetch_Metadata\Policy\Default_Navigation_Isolation_Policy;
+use Google\WP_Fetch_Metadata\Policy\Default_Resource_Isolation_Policy;
+use Google\WP_Fetch_Metadata\Policy_Registry;
+use Google\WP_Fetch_Metadata\Enforce_Policies;
+use Google\WP_Fetch_Metadata\Policies;
+use Google\WP_Fetch_Metadata\Policies_Setting;
+use Brain\Monkey\Functions;
+
+/**
+ * Class containing a Enforce_Policies Tests.
+ */
+class Enforce_Policies_Tests extends Integration_Test_Case {
+
+	const DEST = 'HTTP_SEC_FETCH_DEST';
+	const SITE = 'HTTP_SEC_FETCH_SITE';
+	const MODE = 'HTTP_SEC_FETCH_MODE';
+
+	/**
+	 * Tests enforce() when all default policies are enabled and respected.
+	 *
+	 * @since 0.0.1
+	 */
+	public function testEnforceDefaultPoliciesEnabledRespected() {
+		add_filter( 'wp_die_handler', 'wp_die_handler_filter' );
```

Comment:
Please move these to `setUp` and `tearDown`.

Reply:
Fixed.